base_report_docx/report/report_docx.py

```diff
@@ -226,9 +234,17 @@ def _convert_docx_from_template(
             template_path, base64.b64decode(action.template_file.datas))
 
         doc = DocxTemplate(template_path)
+        self.add_barcode_for_doc(doc, context, data)
```

Comment:
not sure we should do it like that: too specific.
This function should be added as a super to the render function I think (in specific module)
cc @victormartinelicocorp

Reply:
Agree, but it will be a big change for the base module and also the module inherited.

Reply:
please do it. Otherwise we wouldn't ask for it

Reply:
Rename to `on_doc_render(doc, context, data)`